components/pricing_modal/content.tsx

```diff
@@ -177,7 +177,8 @@ function Content(props: ContentProps) {
     }
 
     const openPurchaseModal = () => {
-        trackEvent('cloud_admin', 'click_open_purchase_modal');
+        const utcTimeStamp = new Date().toISOString();
+        trackEvent('cloud_admin', 'click_open_purchase_modal', {utcTimeStamp, epochTimeStamp: Date.parse(utcTimeStamp)});
```

Comment:
Should we rename this to upgrade button clicked? Purchase modal can be opened from many places so maybe: 
```suggestion
        trackEvent('cloud_pricing, 'click_upgrade_button')});
```